### PR TITLE
Add `hyperterm-final-say` to packages.

### PR DESCRIPTION
**Pull request by amio**
`hyperterm-final-say` let user's configuration overrides plugins'.
https://www.npmjs.com/package/hyperterm-final-say

## Checklist:

- [x] The title for my package or theme uses its npm title (hyperterm-final-say)
- [x] The link for my package or theme uses the npmjs.com link (https://npmjs.com/package/hyperterm-final-say)
- [x] There is a visual representation of what my plugin does in the repo.
- [x] **VERY IMPORTANT:** I've written a short (one sentence) description for my package or theme of why it's awesome and deserves to be on the list.

### CODE REVIEW

**Comment by bnb** [comment]
In the package description, can you give a bit more of a description of what this does? 😄 It's not entirely clear right now.

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Like `awesome-hyperterm`? Reach out to me and say *hi* on [Twitter](https://twit
 * [hyperterm-open-devtools](https://www.npmjs.com/package/hyperterm-open-devtools) - Open DevTools for currently showing web page with a hotkey.
 * [hyperterm-close-on-left](https://www.npmjs.com/package/hyperterm-close-on-left) - Positions the close tab button on the left.
 * [hyperterm-sync-settings](https://www.npmjs.com/package/hyperterm-sync-settings) - Easy way to backup and restore HyperTerm settings to Github.
+* [hyperterm-final-say](https://www.npmjs.com/package/hyperterm-final-say) - Let user's configuration overrides plugins'.
```
Comment (bnb) [inline README.md]: `overrides` > `override`

Comment (amio) [inline README.md]: 👌

**Comment by amio** [comment]
I'm not sure how to describe it simple & clearly, this plugin should be put at the last position of plugin list, then it will re-apply user's configuration, to resolve this problem:
> If user installed a theme(a theme has set cursorColor), he won't be able to modify cursorColor within `~/.hyperterm.js`, since plugins have higher priority than `~/.hyperterm.js`

Could you help me refine the description to be more clearly? Appreciate!

FYI,
the code: [hyperterm-final-say's index.js](https://github.com/amio/hyperterm-final-say/blob/master/index.js)
the problem it try to solve: zeit/hyperterm#292

**Comment by bnb** [comment]
How about:

```
The plugin allows user-set overrides of any plugin or theme settings applied on top of the defaults `./.hyperterm.js`. 
```

**Comment by bnb** [comment]
Oops 😅 Didn't mean to close.

**Comment by amio** [comment]
Thanks! I have update the description, See if it's OK :D

**Comment by bnb** [comment]
@amio Excellent! Merged!

**Pull request merged**